docs/matches/basics.md

```diff
@@ -467,6 +505,16 @@ and store all your images there. Let's say I stored the `cat.png` image. We can
 
 At this point, if you type `:nested` you'll see `This is a nested match` appear.
 
+## Keyboard Triggers
+
+Whilst Espanso is able to respond to CTRL-key triggers, it doesn't support combinations with ALT or META etc.. The following works:
+```yml
+  - trigger: "\x05" # <ctrl-e>
```

Comment:
There's a chance this behavior is platform-dependent and won't work across all operating systems

Reply:
Perhaps I should change that to something like: 
“Whilst Espanso is able to respond to CTRL-key triggers, this may be platform dependent, and it doesn't support combinations with ALT or META etc.. The following example works on Linux:”

Reply:
Sounds right smeech!

Reply:
> There's a chance this behavior is platform-dependent and won't work across all operating systems

It works here in Linux and Windows 10.

Reply:
Amended files uploaded.

Reply:
Great! when a thing is "kinda resolved" you can click this button right down the conversation to hide the comments that are not relevant to keep a discussion.

![image](https://github.com/espanso/website/assets/30557287/cb76c5c0-2745-4092-940e-0fc861e444fc)

They can be read if needed, but are folded so new people doesn't read conversations that were resolved unnecesary